README.md

```diff
@@ -1,10 +1,86 @@
 # Data Visualization in Python
+
+This repo contains all the various ways to implement python libraries for visualization of data in python
+=========================================================================================================
+
 <p align="center">
   <img src="https://github.com/Debaditya-Som/Data-Visualization-in-Python/assets/121785700/d40a65d0-ec85-462a-9475-4982a375d067" width=50%/ alt="Data Visualization Image">
 </p>
 
 Data visualization is a powerful tool for understanding and communicating insights from data. This repository aims to provide a comprehensive collection of Python libraries and techniques.
 
+## 001_getting_started
+
```

Comment:
I have created a separate folder for the outputs this part in no longer required